pipeline_dp/accumulator.py

```diff
@@ -42,6 +46,13 @@ def add_value(self, value):
     """
         pass
 
+    def _check_accumulator(self, accumulator: 'Accumulator'):
```

Comment:
Maybe _check_is_mergeable? or _can_merge?

Reply:
`_check_mergeable` is good enough?